docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -728,7 +730,13 @@ Default stratifications to all observers should include scenario and input draw.
       * Ultrasound coverage
       * Azithromycin coverage
       * Misoprostol coverage
+      * Hemoglobin screening coverage
+      * Ferritin screening coverage
+      * True hemoglobin exposure (dichotomous,  'yes' if truly low hemoglobin and 'no' if truly adequate hemoglobin)
+      * Test hemoglobin exposure (dichotomous, 'yes' if tested low hemoglobin,'no' if tested adequate hemoglobin, N/A if not tested)
+      * Ferritin status (dichotomous, 'yes' if low ferritin, 'no' if adequate ferritin, N/A if not tested)
```

Comment:
Let's just make the values here "low" and "adequate" rather than yes/no so that it's a bit easier to interpret